tests/ha/clvm.pm

```diff
@@ -29,15 +29,15 @@ sub run() {
         type_string "echo wait until clvmd resource is created\n";
     }
     else {
-        assert_script_run q(EDITOR="sed -ie '$ a primitive clvmd ocf:lvm2:clvmd op monitor interval=60 timeout=60'" crm configure edit);    #create clvmd primitive
-        assert_script_run q(EDITOR="sed -ie 's/^\\(group base-group.*\\)/\1 clvmd/'" crm configure edit);                                   #add clvmd to base-group
-        sleep 10;                                                                                                                           #wait to get clvmd running on all nodes
+        assert_script_run q(EDITOR="sed -ie '$ a primitive clvmd ocf:heartbeat:clvm op monitor interval=60 timeout=60'" crm configure edit);    #create clvmd primitive
```

Comment:
Maintenance are likely to want to reuse our tests. is `ocf:heartbeat:clvm` available in SLE 12 GA and SP1? If no, then maybe this is a case of an `if` to check for the version and use the right clvmd RA for the right version.